src/doc_builder/convert_rst_to_mdx.py

```diff
@@ -394,6 +396,34 @@ def parse_rst_docstring(docstring):
             lines.insert(idx, "</parameters>\n")
             idx += 1
 
+        # Raises section
```

Comment:
I can't comment on line 385 but the test there will need to include the `_re_raises` as well now, same for the test at line 440.

Might be worth grouping both the "Returns" and "Raises" in the same re pattern and then have subtest there? I'll let you decide.